src/Components/Catalog/Main/Main.js

```diff
@@ -14,17 +17,29 @@ function Main(){
     useEffect(() => {
         const getFilms = async () => {
             setLoading(true)
-            const res = await axios.get('http://www.omdbapi.com/?i=tt3896198&apikey=fb59bcb5')
-            setFilms(res.data)
+            const res = await axios.get('http://www.omdbapi.com/?s=spiderman&apikey=fb59bcb5')
+            console.log(res.data.Search)
```

Comment:
we can you console.log only during development. Remove it please. In this project you can leave console.log only when it is necessary to handle server errors (i mean catch). Later we will also add functionality for handling errors.